WebDriverAgentLib/Categories/XCUIApplication+FBHelpers.m

```diff
@@ -108,4 +108,19 @@ + (NSDictionary *)accessibilityInfoForElement:(XCElementSnapshot *)snapshot
   return info;
 }
 
+-(void) fb_resolve
+{
+  if (self.fb_isActivateSupported) {
+    
+    if(self.state != XCUIApplicationStateRunningForeground)
+    {
+      [self activate];
+    }
+  }
```

Comment:
please format the parentheses according to the common code style

Reply:
No probs, will fix as soon as I can 😉